packages/api/server.js

```diff
@@ -61,75 +45,54 @@ app.get('/api/repos', async (req, res) => {
   }
 
   const jsonObject = { repoNames: repoNames };
-  // TODO allow the requester to give search/filter criterion!
+
   res
     .status(200)
     .send(jsonObject)
     .end();
 });
 
-app.get('/api/auth', (req, res) => {
+app.post('/api/repo/:orgname/:reponame/test', async (req, res) => {
```

Comment:
I tried making it DELETE as we discussed, but found it much easier to redirect from a POST endpoint than from DELETE.

Reply:
I think `DELETE` is probably the correct verb for this type of operation, the basic idea of REST is this:

* `GET`: I'm fetching something.
* `POST`: Create something.
* `PUT`: update something.
* `DELETE`: I intend to delete something.

I did a little test, and this should work:

If you switch `app.post('/api/repo/:orgname/:reponame/test', async (req, res) => {` to
`app.delete('/api/repo/:orgname/:reponame/test', async (req, res) => {`.

```js
    it.only('generates a GitHub redirect', async () => { 
      stubs.push(sinon.stub(repo, 'storeTicket').returns(true));
      const resp = await fetch('http://0.0.0.0:3000/api/repo/my-org/my-repo/test', {
        redirect: 'manual',
        method: 'DELETE'
      });
      assert(resp.headers.get('location').includes('github.com/login/oauth'));
    })
```

I think another thing we should probably change, is that for the route that deletes a specific test we'll want to do:

```
/api/repo/:orgname/:reponame/test/:testid
```

since we'll need to know the test ID too 👍